cmd/user-agent/src/pages/chapi/wallet/didmgmt/didManager.js

```diff
@@ -97,14 +97,18 @@ export class DIDManager extends KeyValueStore {
             }
         )
 
-        const t = await new this.trustblocAgent.Framework(this.trustblocStartupOpts)
+        if (this.trustblocStartupOpts.sdsServerURL) {
+            const t = await new this.trustblocAgent.Framework(this.trustblocStartupOpts)
 
-        // Save DID to persistent storage
-        await t.didclient.saveDID({
-            name: name,
-            signType: signType,
-            did: did
-        })
+            // Save DID to persistent storage
+            await t.didclient.saveDID({
```

Comment:
Are we not even storing in indexeddb if `sds` is not configured ? Or is there another save call to indexeddb ?

Reply:
Yeah, there's another save call to local browser storage just above this - it's cut off by the diff.